packages/puppeteer/test/axePuppeteer.test.ts

```diff
@@ -694,6 +694,21 @@ describe('AxePuppeteer', function () {
     });
   });
 
+  describe('axe.finishRun errors', () => {
```

Comment:
Can you adopt the tests as written?

https://github.com/dequelabs/axe-test-fixtures/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R304